src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: $card-size;
+
+  background: #fff;
+  border: 1px solid $color-light;
+  border-radius: 5px;
+  padding: 32px $half-size $half-size;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #{$color-dark};
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #{$color-lightgrey};
+    margin-bottom: $half-size;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+
+    color: #{$color-dark};
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+
+    margin-bottom: $half-size;
+  }
+
+  &__price-text {
+    color: #{$color-lightgrey};
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+  }
+
+  &__prise-sum {
+    color: #{$color-dark};
+    font-size: $half-size;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__button-link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+
+    font-size: 14px;
+    font-weight: 700;
+    line-height: $half-size;
+
+    text-decoration: none;
+    text-transform: uppercase;
+    color: #{$color-light};
```

Comment:
you should use `#fff` color for link by default. Also you should use the color for link background on hover.